pkg/util/exec/exec_test.go

```diff
@@ -43,3 +50,91 @@ func Test_assertErrorType(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractExitCode(t *testing.T) {
+	make_execExitError := func(retcode int) *exec.ExitError {
```

Comment:
```suggestion
	makeExecExitError := func(retcode int) *exec.ExitError {
```

Missing Python? 😛

Reply:
TBH, I think it might be better as-is. Go suggests against underscores, but I see them used in tests more often. I think this translates more clearly as `make exec.ExitError` if it's `make_execExitError` than `makeExecExitError`.